lib/active_admin/filters/humanized.rb

```diff
@@ -9,8 +9,26 @@ def initialize(param)
         @value = param[1]
       end
 
+      def related_class
+        @related_class ||=
+            begin
```

Comment:
@ccallebs, I want to refactor this in future , to make it not dependent from from ```@body```, and inject active_admin_config and filters  to this part of  ActiveAdmin to make it work always and not to guess related_class.  However now I agree with @momolog, it is better to make it work not in all cases now and fix it in future, because I have no free time at the moment.